server/Cargo.toml

```diff
@@ -34,6 +34,9 @@ authors = ["Aliaksei Bialíauski <aliaksei.bialiauski@hey.com>", "Ivanchuk Ivan
 [lib]
 path = "src/lib.rs"
 
+[lints.rust]
+missing_docs = "warn"
```

Comment:
@h1alexbel I suggesting to change it to `error` and fail build; warnings are quite controversial topic, but imho better to have two states: `passed` and `failed`, warnings create uncertainty by pointing out an error and allowing the build to succeed. I mean, it's like we're saying it's a bad thing to do, but we're gonna let you merge the PR, wdyt?

Reply:
@l3r8yJ good point in terms of passing/failing build. On our CI we have `RUSTFLAGS: "-Dwarnings"` env variable that should fail the build in case of any warnings generated by `cargo`, and PR won't be merged. However, locally often we don't have such flags, therefore would lead to confusion of a contributors in terms of local builds (passes even with warnings) and CI builds (fails on warnings). Error marker should solve this problem.